rules/prefer-includes.js

```diff
@@ -12,13 +12,12 @@ const isIndexOf = node => {
 
 const isNegativeOne = (operator, value) => operator === '-' && value === 1;
 
-const getSourceCode = (context, node) => (
-	context.getSourceCode().getText(node)
-);
-
 const report = (context, node, target, pattern) => {
-	const targetSource = getSourceCode(context, target);
-	const patternSource = getSourceCode(context, pattern);
+	const sourceCode = context.getSourceCode();
+	const memberExpressionNode = target.parent;
+	const dotToken = sourceCode.getTokenBefore(memberExpressionNode.property);
+	const targetSource = sourceCode.getText().slice(memberExpressionNode.start, dotToken.start);
```

Comment:
As found in #272, `start` is not compatible with TS parser
```suggestion
	const targetSource = sourceCode.getText().slice(memberExpressionNode.range[0], dotToken.range[0]);
```